integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -558,4 +591,21 @@ public void createLanguageSubjectObjects() throws Exception {
                 "title", "Cherokee Language Admin", "languageTerm", "chr",
                 "topic", "UNC", "readGroup", "everyone"));
     }
+
+    public void createWorkAndFileObjects() throws Exception {
```

Comment:
It was a little unexpected that this method creates the admin unit and collection, rather than depending on the standard method for creating the structure. I'm fine with this, it should make the test a bit faster, but could you add a comment to the method to indicate that it creates the full structure, rather than just the work and file objects mentioned in the method name?